src/modules/StorageProposal.rb

```diff
@@ -4440,8 +4440,11 @@ def analyse(partitions)
 
       end
 
+
       def modify(partitions)
 
+        Builtins.y2milestone( "Post-processing partitions..." )
```

Comment:
For the new code we should prefer to use the more Ruby way for logging, see http://www.rubydoc.info/github/yast/yast-ruby-bindings/master/Yast/Logger.